docs/01_nodeos/03_plugins/producer_plugin/index.md

```diff
@@ -92,9 +92,10 @@ Config Options for eosio::producer_plugin:
                                         transactions in any block before 
                                         returning to normal transaction 
                                         processing.
-  --incoming-defer-ratio arg (=1)       ratio between incoming transations and 
+  --incoming-defer-ratio arg (=1)       ratio between incoming transactions and 
                                         deferred transactions when both are 
-                                        exhausted
+                                        exhausted                                        
+                                                                            
```

Comment:
`ratio between incoming transactions and deferred transactions when both are queued for execution`

Reply:
See #8691

Reply:
Updated for the code change